rotkehlchen/chain/optimism/modules/hop/constants.py

```diff
@@ -0,0 +1,64 @@
+from typing import Final
+
+from rotkehlchen.chain.evm.constants import ZERO_ADDRESS
+from rotkehlchen.chain.evm.decoding.hop.constants import HopBridge
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants.assets import (
+    A_DAI_OPT,
+    A_ETH,
+    A_HOP_OPT,
+    A_RETH_OPT,
+    A_SNX_OPT,
+    A_SUSD_OPT,
+    A_USDC_OPT,
+    A_USDT_OPT,
+)
+
+ETH_BRIDGE: Final = string_to_evm_address('0x83f6244Bd87662118d96D9a6D44f09dffF14b30E')
+ETH_AMM_WRAPPER: Final = string_to_evm_address('0x86cA30bEF97fB651b8d866D45503684b90cb3312')
+
+USDC_BRIDGE: Final = string_to_evm_address('0xa81D244A1814468C734E5b4101F7b9c0c577a8fC')
+USDC_AMM_WRAPPER: Final = string_to_evm_address('0x2ad09850b0CA4c7c1B33f5AcD6cBAbCaB5d6e796')
+
```

Comment:
have you checked if there is a contract that works as a registry with the information about this contracts?

Reply:
No mention of any registry in their mainnet addresses file.
https://github.com/hop-protocol/hop/blob/d842c19bacff7dbe0b2fda71763f5462a943a146/packages/sdk/src/addresses/mainnet.ts

There is something called `IBonderRegistry`, but it just has `isBonderAllowed` function.
https://github.com/hop-protocol/contracts/blob/3747b0b41defddc5b3c8ca328057847cd987c2c8/contracts/bridges/L1_Bridge.sol#L97